iml-rabbit/Cargo.toml

```diff
@@ -9,10 +9,10 @@ deadpool-lapin = {git = "https://github.com/bikeshedder/deadpool", default-featu
 futures = "0.3"
 iml-manager-env = {path = "../iml-manager-env", version = "0.3"}
 iml-wire-types = {path = "../iml-wire-types", version = "0.3"}
-lapin = "1.2"
+lapin = {version = "1.2", default-features = false, features = ["rustls"]}
 serde_json = "1"
 thiserror = "1.0"
-tokio-amqp = "0.1"
+tokio-amqp = {version = "0.1", default-features = false}
```

Comment:
Is this `default-features` make it enabled, right?
https://github.com/CleverCloud/lapin/blob/master/Cargo.toml#L16

Reply:
yep